### ISSUE TITLE
The tests summary should distinguish between tests passed and skipped

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#2485]
```
*********************************
********* TESTS SUMMARY *********
*********************************
Tests passed or skipped: 212
Tests failed: 0
```
We should inform on passed and skipped tests separately :)

**Comment by AadishJ** [comment]
made a PR https://github.com/pwndbg/pwndbg/pull/2495 please check 😁

**Comment by disconnect3d** [comment]
Thanks, this has been addressed in #2495! :)
